addons/l10n_pe_website_sale/controllers/main.py

```diff
@@ -0,0 +1,79 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
+
+class L10nPEWebsiteSale(WebsiteSale):
+    def _get_mandatory_fields_billing(self, country_id=False):
+        """Extend mandatory fields to add new identification, responsibility
+        city_id and district fields when company is Peru"""
+        res = super()._get_mandatory_fields_billing(country_id)
+        if country_id == request.env.ref("base.pe").id:
```

Comment:
```suggestion
        if country_id.code = 'PE':
```

Reply:
If I do what was suggested I get the following error: 
`if country_id.code == 'PE':
AttributeError: 'int' object has no attribute 'code'`

`country_id` is not record but simply and int of the country's id.